tests/test-cases/Group9-VIC-Admin/9-01-VICAdmin-ShowHTML.md

```diff
@@ -22,3 +22,68 @@ This test requires that a vSphere environment be running and available
 * VICadmin responds with a log file indicating VCH init has begun reaping processes
 * VICadmin responds with log file indicating docker personality service has started
 * VICadmin responds with a ZIP file containing at a minimum the vmware.log file from the new container
+
+#Unauthenticated Test Cases
```

Comment:
Nit: this doesn't render the markdown heading correctly. `# Unauthenticated Test Cases` is the correct format. Bonus points for updating all headings in this file 😃